admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatter.vue

```diff
@@ -39,17 +39,34 @@
         @applyCustomization="applyCustomization"
       >
       </currency-modal>
+
+      <modal
+        confirmation
+        v-if="showConfirmationModal"
+        @confirm="applyResetLanguage"
+        @close="closeConfirmationModal"
+        :modalTitle="modalTitle"
```

Comment:
```suggestion
        :modalTitle="$t('modal.confirm.title')"
```
No need to use computed here, it's wasting memory

Reply:
Oh right, then I guess it's the same for all other computed fields `cancelLabel`, `confirmLabel`, ...

Reply:
Yes :)

Reply:
Done. thanks for the tip